config/initializers/patient_helper.rb

```diff
@@ -0,0 +1,5 @@
+PATIENT_HELPER_FILES = {
```

Comment:
This does not appear to actually set this as a global variable. When calling something like `address_timezone_offset` we get an uninitialized constant error. Seems like the syntax should be updated to be `::PATIENT_HELPER_FILES` instead.

Reply:
You may need to restart your rails console (if you pulled this down while you had it running). This is definitely the right way to do this, and `PATIENT_HELPER_FILES` should be accessible everywhere.

Reply:
@holmesie does this work for you locally?

Reply:
Hmm, I swore I had restarted (since I did after testing that other syntax and it worked fine as well), but I guess I did not. Seems to actually be working fine. So nevermind!

Reply:
```
irb(main):001:0> PATIENT_HELPER_FILES.nil?
=> false
irb(main):002:0> PATIENT_HELPER_FILES.keys
=> [:state_names, :states_with_time_zone_data, :languages]
irb(main):003:0> Patient.last.address_timezone_offset
  Patient Load (0.6ms)  SELECT `patients`.* FROM `patients` ORDER BY `patients`.`id` DESC LIMIT 1
=> "-05:00"
```